packages/siopv2-oid4vp-op-auth/src/link-handler/index.ts

```diff
@@ -0,0 +1,49 @@
+import { LinkHandlerAdapter } from '@sphereon/ssi-sdk.core'
+import { IMachineStatePersistence, interpreterStartOrResume } from '@sphereon/ssi-sdk.xstate-machine-persistence'
+import { IAgentContext } from '@veramo/core'
+import { Loggers, LogMethod } from '@sphereon/ssi-types'
+import { GetMachineArgs, IDidAuthSiopOpAuthenticator, LOGGER_NAMESPACE, Siopv2MachineInterpreter, Siopv2MachineState } from '../types'
+
+const logger = Loggers.DEFAULT.options(LOGGER_NAMESPACE, { methods: [LogMethod.CONSOLE, LogMethod.DEBUG_PKG] }).get(LOGGER_NAMESPACE)
```

Comment:
Do not set options from the SDK

Reply:
I removed the methods everywhere but in the .test.ts files last week